helm/templates/controller/statefulset.yaml

```diff
@@ -45,13 +45,19 @@ spec:
               exitCode=1
               i=0
               while [ $i -le {{ .Values.zookeeper.retries }} ]; do
-                bin/zookeeper-shell.sh ZooKeeper -server {{ include "zookeeper.url" . | quote }} create {{ .Values.zookeeper.path | quote }} ""
+                bin/zookeeper-shell.sh ZooKeeper -server {{ include "zookeeper.url" . | quote }} ls {{ .Values.zookeeper.path | quote }}
```

Comment:
new image returns error if the node already exist in zookeeper.
```
WatchedEvent state:SyncConnected type:None path:null
Node already exists: /pinot
[2022-10-20 16:45:45,596] ERROR Exiting JVM with code 1 (org.apache.zookeeper.util.ServiceUtils)
```

updated script to first check the node before creating it.